simulator/host_datastore_system.go

```diff
@@ -83,11 +84,28 @@ func (dss *HostDatastoreSystem) add(ctx *Context, ds *Datastore) *soap.Fault {
 	parent := hostParent(dss.Host)
 	Map.AddReference(ctx, parent, &parent.Datastore, ds.Self)
 
-	browser := &HostDatastoreBrowser{}
-	browser.Datastore = dss.Datastore
-	ds.Browser = Map.Put(browser).Reference()
+	if Map.Get(ds.Self) == nil {
+		browser := &HostDatastoreBrowser{}
+		browser.Datastore = dss.Datastore
+		ds.Browser = Map.Put(browser).Reference()
 
-	folderPutChild(ctx, folder, ds)
+		var capacity units.ByteSize
+		b := os.Getenv("VCSIM_DATASTORE_CAPACITY")
```

Comment:
Can you please document `"VCSIM_DATASTORE_CAPACITY"` in the README?

Reply:
Not sure we should use an env var here at all.  Tempted to leave it hardcoded as `1TB` (give or take) and find a better way to configure the default if we find we need to.  Any thoughts @protochron ?

Reply:
I'm fine with a hardcoded value, though maybe use something like `10TB` and call it a day?